CHANGELOG.rst

```diff
@@ -10,8 +10,14 @@ Changelog
 0.25
 ====
 
+0.25.2
+------
+Fixed
+^^^^^
+- Remove sensitive query parameters from debug logging to prevent exposure of passwords, tokens, and personal data (#1996)
```

Comment:
Want this too be added to an `## [Unreleased]` section? [Keep A Changelog](https://keepachangelog.com/en/1.1.0/) style?

Reply:
I've added my change to that new section.

Reply:
[814974b](https://github.com/tortoise/tortoise-orm/pull/1997/commits/814974b9b66664562fd9e20e0fd322d1f2c3cab3)